packages/account/src/Sections/Security/LoginHistory/login-history.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Loading, Table, Text, ThemedScrollbars } from '@deriv/components';
+import { isMobile, isDesktop } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { TData } from '@deriv/api/types';
+import { useLoginHistory } from '@deriv/api';
+import LoadErrorMessage from 'Components/load-error-message';
+
+type TListCell = {
+    title: JSX.Element | string;
+    text: JSX.Element | string;
+    className?: string;
+    is_align_right?: boolean;
+};
+
+type TGetFields = {
+    date: JSX.Element;
+    action: JSX.Element;
+    browser: JSX.Element;
+    ip: JSX.Element;
+    status: JSX.Element;
+};
+
+const getFields = () => ({
+    date: <Localize i18n_default_text='Date and time' />,
+    action: <Localize i18n_default_text='Action' />,
+    browser: <Localize i18n_default_text='Browser' />,
+    ip: <Localize i18n_default_text='IP address' />,
+    status: <Localize i18n_default_text='Status' />,
+});
+
+const LoginHistoryContent = ({ data }: { data: TData }) => {
+    if (isMobile()) {
+        return renderList(getFields(), data);
+    }
+    return renderTable(getFields(), data);
+};
+
+const renderTable = (fields: TGetFields, login_history: TData) => (
```

Comment:
shall we destructure fields object here and use it below without dot notation?
`const {date, action, browser,...} = fields;`